app/celery/tasks.py

```diff
@@ -472,6 +477,9 @@ def update_letter_notifications_statuses(self, filename):
                 current_app.logger.info(
                     'DVLA file: {filename}, notification updated to {status}: {reference}'.format(
                         filename=filename, status=status, reference=str(update.reference)))
+                notifications = dao_get_notifications_by_reference(references=[update.reference])
```

Comment:
We can check if the service has a callback url first, that way we do not need to create a task on the queue.

Reply:
changed in 3594fc.